package.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "device-id-widget",
+  "widgetName": "DeviceIdWidget",
+  "version": "1.0.0",
```

Comment:
This is the first version of the widget and so the version stays. @Andries-Smit do you copy?